//What happens when you run the following program? Why do we get that result?

{
  let foo = 'bar';
}

console.log(foo);

// Results in an error since the foo variable does not exist outside of its block.